examples/angular-cli/src/stories/csf3/input-with-play.stories.ts

```diff
@@ -0,0 +1,28 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { InputComponent } from './sb-input.component';
+
+export default {
+  title: 'CSF3/Input',
```

Comment:
maybe `Core / CSF3 / Input` ?

Reply:
Would "Preview" be a fitting prefix perhaps?

Reply:
- In core I tried to place the stories showing and testing the native functionality of sotrybook
- basics it's more for examples of storybook use 
- addons for addons
..
so if CSF3 is to become something native in storybook main set, I can see it in `core/...` 

but it's up to you, knowing that ;)